tests/transaction.test.ts

```diff
@@ -25,19 +27,39 @@ describe("transaction", () => {
         dependsOn: null,
         nonce: 12345678,
     }
-    const unsigned = new Transaction(body)
+
+    // Correct unsigned transaction
+    const unsigned = new Transaction(correctTransactionBody)
     const unsignedEncoded = Buffer.from('f8540184aabbccdd20f840df947567d83b7b8d80addcb281a71d54fc7b3364ffed82271086000000606060df947567d83b7b8d80addcb281a71d54fc7b3364ffed824e208600000060606081808252088083bc614ec0', 'hex')
 
+    // Delegated transaction
+    const delegated = new Transaction({
+        ...correctTransactionBody,
+        reserved: {
+            features: 1
+        }
+    })
+
+    // Incorrectly delegated transaction
+    const incorrectlyDelegated = new Transaction({
+        ...correctTransactionBody,
+        // Invalid features field
+        reserved: {
+            features: 1,
+            unused: [Buffer.from('0x00', 'hex'), Buffer.from('0x00', 'hex')]
```

Comment:
If you want empty buffer here,  `Buffer.alloc(0)` would be better.

Reply:
Hi thanks for suggestion. Fixed here: https://github.com/vechain/thor-devkit.js/pull/59/commits/501c3cc1b88f0a3a69c8dd0f21cbb867f54d4791